web/packages/teleport/src/Account/Account.test.tsx

```diff
@@ -370,11 +370,13 @@ test('removing an MFA method', async () => {
   cfg.auth.allowPasswordless = true;
 
   await renderComponent(ctx);
-  await user.click(
+  fireEvent.click(
```

Comment:
i don't understand why i had to use fireEvent here... (another file also required a fireEvent)

Reply:
thats funny you bring that up. @gzdunek had the opposite experience and managed to get `userEvent` to work better than `fireEvent`. YMMV i guess 🤷

Reply:
Yeah, in general it's better to use `userEvent` over `fireEvent`, since it's a higher-level API.
If `userEvent` isn't working, the issue must be somewhere else 🤔

Reply:
idk what happened in between me submitting the fix and reverting the fireEvents... reverting the fireEvent back to userEvent just works now.

i won't question it.

Reply:
ohhh!